dev-libs/libgrapheme/libgrapheme-2.0.0.recipe

```diff
@@ -0,0 +1,58 @@
+SUMMARY="A unicode string library"
+DESCRIPTION="libgrapheme is an extremely simple freestanding C99 library providing \
+utilities for properly handling strings according to the latest Unicode standard \
+15.0.0. It offers fully Unicode compliant."
+HOMEPAGE="https://libs.suckless.org/libgrapheme"
+COPYRIGHT="2022 Laslo Hunhold"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="https://dl.suckless.org/libgrapheme/libgrapheme-$portVersion.tar.gz"
+SOURCE_FILENAME="libgrapheme-$portVersion.tar.gz"
+CHECKSUM_SHA256="a6e1f7aec86af029dc183e32ab7a8d531fc00e54c44921dbdffc929aceb90dc6"
+PATCHES="libgrapheme-$portVersion.patchset"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libgrapheme$secondaryArchSuffix = $portVersion
+	lib:libgrapheme$secondaryArchSuffix = 4.0.0 compat >= 4
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libgrapheme${secondaryArchSuffix}_devel = $portVersion
+	devel:libgrapheme$secondaryArchSuffix = 4.0.0 compat >= 4
+	"
+REQUIRES_devel="
+	libgrapheme$secondaryArchSuffix == $portVersion
+	"
```

Comment:
You've declared a devel package but there are no commands(`prepareInstalledDevelLib` or `packageEntries`) in the INSTALL section to put the library symlinks, headers,  and man pages there.  So, the devel package ends up being empty.

Reply:
updated

Reply:
`prepareInstalledDevelLib` still missing ...

Reply:
updated (pardon for leaving that out)